parsing/dialects/intermediate/dialects/dialect1/parserImpls.ftl

```diff
@@ -3397,6 +3397,8 @@ SqlDrop SqlDrop() :
         drop = SqlDropView(s, replace)
     |
         drop = SqlDropFunction(s, replace)
+    |
+        drop = SqlDropProcedure(s)
```

Comment:
nit: While you're here, can you alphabetize all of these SqlDrop clauses?

Reply:
Sure, it's updated now